task-management/src/main/webapp/WEB-INF/gatein-resources.xml

```diff
@@ -379,6 +379,9 @@
       <depends>
         <module>userPopupPlugin</module>
       </depends>
+      <depends>
```

Comment:
no direct dependency, should use extensionRegistry to inject extensible behavior between addons